src/main/java/LoginTest.java

```diff
@@ -20,6 +20,9 @@
     @Test
     void loginTest() throws InterruptedException {
 
+        LoginPage loginPage = new LoginPage();
+        HomePage homePage = new HomePage();
```

Comment:
you should not create pages out of thin air. later we will introduce pageFactory, but for now, please, do not initialize variables ahead of time. 
init login page right before you need it. and do not create Home page with a default constructor here - "loginPage.clickLoginButton()" line in test should return you this page. HomePage homePage = loginPage.clickLoginButton() - like this. imagine browser, user clicks login button - and receives new page. same should be in our auto-tests